content/blog/harbor-at-fosdem-2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Harbor at FOSDEM 22"
+author:
+  name: "Orlin Vasilev"
+description: "Harbor - The Container Registry"
+date: 2022-03-24T12:00:00+04:00
+showPageInfo: true
+---
+
+## Harbor - The Container Registry at FOSDEM 2022!
+
+I'm excited to share that I recently had the opportunity to represent the Harbor community at [FOSDEM 2022](https://fosdem.org/2022/). As a longtime FOSDEM attendee (my 10th-ish year in a row), this was my first time speaking at the conference. It was a dream come time to be accepted to the [Containers devroom track](https://fosdem.org/2022/schedule/track/containers/) and talk with my fellow attendees about the Harbor project.
+​
+As everything during the pandemic this year's FOSDEM was 100% virtual. Next year I hope we can meet each other in person at the venue, but until then below is the link to the recorded talk and slides.
+​
+* [Harbor - The Container Registry - 101](https://fosdem.org/2022/schedule/event/container_harbor/)
+## What is FOSDEM?
+
+```
+FOSDEM == "Free and Open source Software Developers' European Meeting";
+```
+​
+For many non-Europeans, FOSDEM might not sound familiar, it’s the biggest FREE and OpenSource conference held in Europe. FOSDEM stands for Free and Open source Software Developers' European Meeting. Started by Raphael Bauduin in 2000, this conference always happens the first weekend of February in Brussels, Belgium at Université Libre de Bruxelles.
+​
+It’s company and organization sponsored but represents only projects and communities not companies, which makes it unique in the Open Source world these days! As such it’s always packed with folks from around the globe, some representing their projects others just for the beer and waffles.
+
+![We are full](https://upload.wikimedia.org/wikipedia/commons/0/00/FOSDEM_2015_full_sign_1.jpg)
+​
+## Connect with the Harbor team in 2022
+​
+Harbor is a [healthy](https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1) and vibrant Graduated project at CNCF. We’ve seen growth in our community and superb contributions in the form of issues, code, and project feedback. If you'd like to connect with us, the Harbor team currently plans to attend the following conferences and events this year.
+​
+### KubeCon EU 2022 - Valencia
```

Comment:
Maybe add links to these conference schedules? or where folks can get more information?

Reply:
good idea done!